flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/spec/FlinkDeploymentSpec.java

```diff
@@ -41,5 +41,4 @@
     private JobManagerSpec jobManager;
     private TaskManagerSpec taskManager;
     private JobSpec job;
-    private Map<String, String> logging;
```

Comment:
Do we remove the `logging` field because it is not supported now?

Reply:
yes, also we discussed it with @gyfora that map is not the ideal form here for the logging, we'll have to reintroduce it in another form to support log4j, logback.. configs

Reply:
Great.